app/views/users/show.html.erb

```diff
@@ -2,19 +2,27 @@
   <%= gravatar_for @user %>
   <%= @user.name %>
 </h1>
-<% if @user.posts.any? %> 
+<% if @user.posts.any? %>
 	<h3> Posts (<%= @user.posts.count %>)</h3>
-	  <ol> 
-		  <% @user.posts.each do |post| %> 
-        <li> 
-          <%= link_to post.title, post.url %> 
+	  <ol>
+		  <% @user.posts.each do |post| %>
+        <li>
+          <%= link_to post.title, post.url %>
           <br>
           <br>
           Posted <%= time_ago_in_words(post.created_at) %> ago.
         </li>
```

Comment:
it's possible you're mixing tabs and spaces, if that's the case i'd recommend just using spaces, you can typically set up your text editor to default to one